app/components/__tests__/AdvancedSearchPage.test.js

```diff
@@ -18,7 +18,7 @@ const setup = () => {
   };
 
   const standard = {
-    filter: {},
+    filter: 'ibn',
```

Comment:
This is a test file so it doesn't matter much, but the correct filter would be `iba`, just for information.

Reply:
ok :+1: